### ISSUE TITLE
documentation/feature: using router for resolving local addresses

### ISSUE THREAD

**Issue by maxgrass** [kkk669/DNSecure#52]
The core functionality works.

Now in my intranet say, `*.my.local` there are nodes (e.g. `node1.my.local`). The router is usually a DNS-server for those local nodes.

- Is it currently possible (how?) to add an "exclusion" and use the Router DNS to resolve those hostnames on the local network?

**Comment by kkk669** [comment]
No, it's not possible at this time. We need to fix #47 in order to implement that feature.

**Comment by holygrolli** [comment]
any chance to implement this feature? when #47 would be implemented then the app decided on each dns request to which upstream dns to forward to?

**Comment by yurykk** [comment]
@kkebo Hi

Are there any plans to implement that ? It seems to be an very important feature.

**Comment by kkebo** [comment]
I haven't fully understood the behaviors of APIs listed in #47 yet, so it might take a little time. Pull requests are always welcome.

**Comment by kkebo** [comment]
#47 has been resolved in 1.6.0, so this issue can be resolved by using On-Demand Rules.
